packages/x-date-pickers/src/tests/describeValue/testPickerOpenCloseLifeCycle.tsx

```diff
@@ -187,14 +215,19 @@ export const testPickerOpenCloseLifeCycle: DescribeValueTestSuite<any, 'new-pick
       userEvent.keyPress(document.activeElement!, { key: 'Escape' });
       expect(onChange.callCount).to.equal(1);
       expect(onAccept.callCount).to.equal(1);
-      // TODO: Support range
-      expect(onAccept.lastCall.args[0]).toEqualDateTime(newValue as any);
+      if (pickerParams.type === 'date-range') {
+        newValue.forEach((value, index) => {
+          expect(onChange.lastCall.args[0][index]).toEqualDateTime(value);
+        });
+      } else {
+        expect(onChange.lastCall.args[0]).toEqualDateTime(newValue as any);
+      }
       expect(onClose.callCount).to.equal(1);
     });
 
     it('should call onClose when clicking outside of the picker without prior change', () => {
-      // TODO: Fix this test and enable it on mobile
-      if (pickerParams.variant === 'mobile') {
+      // TODO: Fix this test and enable it on mobile and date-range
+      if (pickerParams.variant === 'mobile' || pickerParams.type === 'date-range') {
```

Comment:
As discussed with @flaviendelangle —it's time consuming and not trivial to track down why clicking on body in `jsdom` does not correctly close date range picker 🤷